.github/workflows/deploy-storybook.yml

```
@@ -14,10 +14,8 @@ jobs:
       - name: build
         run: |
           yarn install --frozen-lockfile
-          yarn validate
           yarn build:storybook:ci
       - name: deploy
+        run: yarn deploy:storybook -- --ci
```

i think yarn passes these through automatically

```suggestion
        run: yarn deploy:storybook --ci
```